bot/match/processes/player_picking.py

```diff
@@ -50,10 +52,15 @@ async def init(self):
         """ Init the match channel, ping players, find two captains \
             and ask them to start picking players.
         """
-        # Ready for players to pick
-        ctx = self.interaction_handler.get_new_context(self.match.channel)
-        await disp.MATCH_SHOW_PICKS.send(ctx, self.match.teams[0].captain.mention,
-                                         match=self.match.proxy)
+
+        if (len(self.players) == 0):
+            log.debug(f"there are 0 players, meaning this is a 2 person lobby with 2 captains already!")
```

Comment:
Shouldn't we just enforce 4 min players?

Reply:
being able to test with 2 accounts was really nice. i don't see a reason to not use this

Reply:
Are you sure it's not breaking something else?

Reply:
i successfully ran both a 2 player, and a 4 player (force started) lobby. if you really are worried about this, we can remove it, but it makes testing harder